docs/standard/commandline/command-line-configuration.md

```diff
@@ -0,0 +1,55 @@
+---
+title: How to configure the parser in System.CommandLine
+description: "Learn how to configure the parser in System.CommandLine."
+ms.date: 16/06/2025
+no-loc: [System.CommandLine]
+helpviewer_keywords:
+  - "command line interface"
+  - "command line"
+  - "System.CommandLine"
+ms.topic: how-to
+---
+
+# How to configure the parser in System.CommandLine
+
+[!INCLUDE [scl-preview](../../../includes/scl-preview.md)]
+
+<xref:System.CommandLine.CommandLineConfiguration> is a class that provides properties to configure the parser. It is an optional argument for every `Parse` method, such as <xref:System.CommandLine.Command.Parse> or <xref:System.CommandLine.Parsing.CommandLineParser.Parse>. When it is not provided, the default configuration is used.
+
+Every <xref:System.CommandLine.ParseResult> instance has a <xref:System.CommandLine.ParseResult.Configuration> property that returns the configuration used for parsing.
+
+## Standard output and error
```

Comment:
This article also isn't really a how to in the traditional sense. It just sort of plops a single snippet of code down and has some sub sections on what I assume are parameter options? But these descriptions don't really explain the options. For example, what does **ResponseFileTokenReplacer** actually do? Why would someone use it?

I would rewrite this to have a better "how to" approach:

- Configure the parser to do X
- Configure the parser to do Y
- Additional configuration options